CSharpMath.Editor.Tests/CaretTests.cs

```diff
@@ -177,19 +176,51 @@ public class CaretCanStartAndStopBlinking {
       Assert.Equal(MathKeyboardCaretState.Shown, keyboard.CaretState);
     }
   }
+  public class DefaultPlaceholderSettings {
+    public const string ActiveNucleus = "■";
+    public const string RestingNucleus = "□";
+    public static readonly System.Drawing.Color? ActiveColor = null;
+    public static readonly System.Drawing.Color? RestingColor = null;
```

Comment:
Unsolved mystery: reference type constants can be declared but have to be null, while nullable value type constants can't even be defined at all. I couldn't find an explanation online.